app/core/config.py

```diff
@@ -0,0 +1,10 @@
+import os
+from dotenv import load_dotenv
+
+# Load .env file
+load_dotenv()
+
+# Fetch environment variables
+MONGO_URI = os.getenv("MONGO_URI")
```

Comment:
rather than having this file ? 
Isn't this doable with just the environment variables.. managed by IDE?

Reply:
this way.. we also won't need the .env file